zipkin-server/src/main/java/zipkin2/server/internal/throttle/ThrottledCall.java

```diff
@@ -38,6 +40,13 @@
  * @see ThrottledStorageComponent
  */
 final class ThrottledCall extends Call.Base<Void> {
+  /**
+   * Rather than flooding when concurrency reached, return the same instance. The path to this is
```

Comment:
I'm not sure what flooding means here, so the comment takes much longer to parse. Is it more like `Reduce allocations when concurrency reached by always returning the same instance. This is only thrown in one location so the stack trace isn't useful and we clear it to keep logs down.`?

Reply:
basically what you said is what I mean :P